global.json

```
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "10.0.100-preview.7.25322.101"
+    "version": "10.0.100-preview.7.25372.107"
   },
   "tools": {
-    "dotnet": "10.0.100-preview.7.25322.101",
+    "dotnet": "10.0.100-preview.7.25372.107",
```

```suggestion
    "dotnet": "10.0.100-preview.7.25380.108",
```
See https://github.com/dotnet/sdk/issues/49786